src/transforms/stdnames.jl

```diff
@@ -0,0 +1,59 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    StdNames(S)
+
+StdNames transform standardizes the column names.
+"""
+struct StdNames <: Stateless
+    stdmethod::Symbol
+end
+
+isrevertible(::Type{StdNames}) = true
+
+function apply(transform::StdNames, table)    
+    if transform.stdmethod == :camel
+        _camel(table)
+    elseif transform.stdmethod == :snake
+        _snake(table)
+    else
+        _upper(table)
+    end
+end
+
+function revert(::StdNames, newtable, cache)
+    names = Tables.columnnames(newtable)
+    namesdict = Dict(zip(names, cache))
+    newtable |> Rename(namesdict)
```

Comment:
Try to use the same variable names used in other transforms.

Reply:
I assumed you were referring to `namesdict` variable.